setup.py

```diff
@@ -54,7 +54,7 @@ def get_console_scripts() -> List[str]:
     "gptq": ["optimum>=1.17.0", "auto-gptq>=0.5.0"],
     "awq": ["autoawq"],
     "aqlm": ["aqlm[gpu]>=1.1.0"],
-    "vllm": ["vllm>=0.4.3,<=0.6.0"],
```

Comment:
The latest version of vllm should be v0.6.2, the upper bound cannot be a non-exist version